### DISCUSSION TITLE
Rugpull Retribution license?

### DISCUSSION THREAD

**Discussion by oenu** [opentffoundation/manifesto#760] [Ideas]
Would be funny and cool if the license was updated to prevent contributions from being pulled into Hash's fork (if not already covered by the license)

Could be a nice trend

**Comment by anton-yurchenko** [comment]
It sounds like [GPL](https://en.wikipedia.org/wiki/GNU_General_Public_License) can do the trick, but I don't know whether **BUSL** counts as an _equivalent license_:
> any derivative work must be distributed under the same or equivalent license terms

Certainly, someone here has a deep knowledge of this topic 😉

**Comment by innovate-invent** [comment]
I am wondering if TF Providers should start changing their licenses to block use by non-open TF releases.

**Comment by DicsyDel** [comment]
We have only one goal which is ensure openness of terraform by making OpenTF successful. If Hashi wants to pull contributions, they're welcome to do so. We have no intentions or desire to start a new licensing war.
